apps/judicial-system/web/src/routes/Shared/Statement/Statement.tsx

```diff
@@ -0,0 +1,228 @@
+            </Text>
+          )}
+        </Box>
+
+        {user && (
+          <>
+            <Box component="section" marginBottom={5}>
+              <SectionHeading
+                title={formatMessage(strings.uploadStatementTitle)}
+                required
+              />
+              <InputFileUpload
+                fileList={displayFiles.filter(
+                  (file) => file.category === appealStatementType,
+                )}
+                accept={'application/pdf'}
+                header={formatMessage(core.uploadBoxTitle)}
+                description={formatMessage(core.uploadBoxDescription, {
+                  fileEndings: '.pdf',
+                })}
+                buttonLabel={formatMessage(core.uploadBoxButtonLabel)}
+                multiple={false}
+                onChange={(files) =>
+                  handleChange(
+                    files,
+                    appealStatementType,
+                    setDisplayFiles,
+                    handleUIUpdate,
+                  )
+                }
+                onRemove={(file) => handleRemove(file, removeFileCB)}
+                onRetry={(file) => handleRetry(file, handleUIUpdate)}
+              />
+            </Box>
+            <Box component="section" marginBottom={10}>
+              <SectionHeading
+                title={formatMessage(strings.uploadStatementCaseFilesTitle)}
+                marginBottom={1}
+              />
+              <Text marginBottom={3}>
+                {formatMessage(strings.uploadStatementCaseFilesSubtitle)}
+              </Text>
+              <InputFileUpload
+                fileList={displayFiles.filter(
+                  (file) => file.category === appealCaseFilesType,
+                )}
+                accept={'application/pdf'}
+                header={formatMessage(core.uploadBoxTitle)}
+                description={formatMessage(core.uploadBoxDescription, {
+                  fileEndings: '.pdf',
+                })}
+                buttonLabel={formatMessage(core.uploadBoxButtonLabel)}
+                multiple={false}
+                onChange={(files) =>
+                  handleChange(
+                    files,
+                    appealCaseFilesType,
+                    setDisplayFiles,
+                    handleUIUpdate,
+                  )
+                }
+                onRemove={(file) => handleRemove(file, removeFileCB)}
+                onRetry={(file) => handleRetry(file, handleUIUpdate)}
+              />
+            </Box>
+          </>
+        )}
+      </FormContentContainer>
+      <FormContentContainer isFooter>
+        <FormFooter
+          previousUrl={previousUrl}
+          onNextButtonClick={() => setVisibleModal('STATEMENT_SENT')}
+          nextButtonText={formatMessage(strings.nextButtonText)}
+          nextIsDisabled={!isStepValid}
+          nextButtonIcon={undefined}
+        />
+      </FormContentContainer>
+      {visibleModal === 'STATEMENT_SENT' && (
```

Comment:
Just curiousity, why is the visibleModal state a string rather than a boolean? Is it so we have the option of easily showing a different type of modal if needed?

Reply:
Yes, that's what I'm thinking!